swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
+from .item import Item
+
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0):
+        self.category = category
+        self.condition = condition
```

Comment:
`Clothing` is inheriting from `Item`. Could we make use of the constructor from `Item` to ensure that any initialization that `Item` does will definitely be applied? Think about what would happen if `Item` added a new parameter to its constructor to track when the item was last used and initialized it within the constructor as
```python
    self.last_used = datetime.now()  # assume we have the necessary imports
```
Since `Clothing` is-an `Item`, it should have all the fields that the parent has. Can we write the derived constructor so that we don't need to update every child class of `Item` whenever `Item` changes?

Consider this for the remaining `item`-derived classes.